### ISSUE TITLE
npm 3 / node 5 issues

### ISSUE THREAD

**Issue by gruppjo** [mwaylabs/generator-m-ionic#277]
**TL;DR;** for now you will have to downgrade to npm 2 and node 4 until `cordova-lib` runs with npm 3 and node 5 respectively.

**Update:** Apparently issues are fixed in `cordova 5.4.0`, will test and report back with updates. http://cordova.apache.org/news/2015/11/04/tools-release.html

There are a couple of issues on the `cordova-lib` repo related to npm 3 and node 5:
- https://issues.apache.org/jira/browse/CB-9687
- https://issues.apache.org/jira/browse/CB-9954
- https://issues.apache.org/jira/browse/CB-9916
- https://issues.apache.org/jira/browse/CB-9935

this is a meta issue for all issues with npm 3 / node 5 of generator-m-ionic:
- https://github.com/mwaylabs/generator-m-ionic/issues/270
- https://github.com/mwaylabs/generator-m-ionic/issues/257
- https://github.com/mwaylabs/generator-m-ionic/issues/264

Read more about the npm update [here](http://blog.npmjs.org/post/122450408965/npm-weekly-20-npm-3-is-here-ish) and [here](https://github.com/npm/npm/releases/tag/v3.0.0).

I did some tests and installed the generator with the following setup:
```sh
$ npm list -g generator-m-ionic
/usr/local/lib
└── generator-m-ionic@1.4.0
$ node -v
v4.2.1
$ npm -v
2.14.7
```
And everything went fine.

However with:
```sh
$ npm list -g generator-m-ionic
/usr/local/lib
└── generator-m-ionic@1.4.0
$ node -v
v5.0.0
$ npm -v
3.3.6
```
the following error did occur:
```sh
Couldn't finish generator: 
CordovaError: Could not find directory: /Users/jonathan/Projects/generator-m-ionic/node_modules/cordova-lib/node_modules/cordova-app-hello-world
```
running `npm install` in `generator-m-ionic/node_modules/cordova-lib` did prevent that error from happening but many files were missing upon creation.
![image](https://cloud.githubusercontent.com/assets/1370779/10913163/15a6153c-824d-11e5-94f5-551cedb4708f.png)

**Comment by gruppjo** [comment]
fixed in 1.4.1

**Comment by gruppjo** [comment]
fixed in 1.4.1
